kernel-switcher/test_in_lxd_vm.py

```diff
@@ -0,0 +1,155 @@
+#!usr/bin/env python3
```

Comment:
typo, also, this file is not executable (`no +x`), this is also probably an oversight

Reply:
ah, thanks!